internal/build/tar.go

```diff
@@ -109,6 +103,10 @@ func tarPath(ctx context.Context, tarWriter *tar.Writer, source, dest string) (d
 		if header.Typeflag == tar.TypeReg {
 			file, err := os.Open(path)
 			if err != nil {
+				// In case the file has been deleted since we last looked at it.
+				if os.IsNotExist(err) {
```

Comment:
iiiiii am not SURE this is the right behavior? because we don't delete it in the container, we just don't touch it...

Reply:
i think you're confused, this is reading the file outside the container